jipdate.py

```diff
@@ -128,6 +128,11 @@ def get_parser():
             default=False, \
             help='Load all Jira issues, not just the once marked in progress.')
 
+    parser.add_argument('-r', required=False, action="store_true", \
```

Comment:
Pipe the output but also when I just want to fill a file without having to exit editor and abort the push to server
Ok for renaming it to -p